examples/js/loaders/GLTFLoader.js

```diff
@@ -696,8 +696,16 @@ THREE.GLTFLoader = ( function () {
 
 		if ( ! loader ) {
 
-			// Assumes that the extension is optional and that a fallback texture is present; ideally we should handle extensionsRequired better
-			return null;
+			if ( json.extensionsRequired && json.extensionsRequired.includes( this.name ) ) {
```

Comment:
For the time being we are still using `.indexOf( str ) >= 0` to avoid needing transpilation for IE11.